src/_posts/2017/2017-04-01-statie-generate-and-refresh.md

```diff
@@ -0,0 +1,154 @@
+---
+layout: post
+title: 'Statie: generate & refresh'
+description: '''
+Make development with <a href="https://github.com/Symplify/Statie">Statie</a> a bit more convenient with a little use of <a href="http://gulpjs.com/">gulp</a> and <a href="https://browsersync.io/">Browsersync</a>. Take a look at how to implement <a href="http://lesscss.org/">Less</a> preprocessor in the building workflow.
+'''
+---
+
+With a rising popularity of static site generators, we can find that there are [many of them][1] already in use. This blog runs on [Statie][2], another such a tool written by [Tomas Votruba](https://www.tomasvotruba.cz/), which differs from the others with [Nette](https://nette.org/) and its templating engine [Latte](https://latte.nette.org/) used in the background.
+
+If you are not aware what is this tool all about let's take a look at a post series about it on [Tomas' blog][3] firstly. It will lead you from the very beginning on how to run your own site upon Statie, so I won't deal with this here.
+
+In the end of the mentioned blog post, we can find a [simple gulp task](https://www.tomasvotruba.cz/blog/2017/02/20/statie-how-to-run-it-locally/#minitip-use-gulp-work-for-you) that runs generate command on every change in the source files. This is surely handy, but you still need to refresh your browser to be able to see your changes. With **Browsersync** we can make this step unnecessary.
```

Comment:
This could have a title "Would you like live reload?" or something like that.
To really sell the value of this post, which I like btw.